packages/components/src/mobile/bottom-sheet/range-cell.native.js

```diff
@@ -71,7 +71,9 @@ class BottomSheetRangeCell extends Component {
 			sliderValue: nextValue,
 		} );
 		onChange( nextValue );
-		onComplete( nextValue );
+		if ( onComplete ) {
+			onComplete( nextValue );
+		}
```

Comment:
My first attempt was to pass the handler in the Spacer block but then I realised that it would require to add it in more blocks so a better solution was to make it optional.